airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -114,12 +119,14 @@ def create_glue_job_config(self) -> dict:
             "ScriptLocation": self.script_location,
         }
         command = self.create_job_kwargs.pop("Command", default_command)
-        execution_role = self.get_iam_execution_role()
+        if not self.role_arn:
+            execution_role = self.get_iam_execution_role()
+            self.role_arn = execution_role["Role"]["Arn"]
```

Comment:
nit. There is no need to save it as instance variable, local variable is enough
```suggestion
            role_arn = execution_role["Role"]["Arn"]
```

Reply:
@vincbeck updated. assigned the role for both cases to a local variable `role_arn`. Let me know if this works. I did not want to put the if/else logic on line #131.